examples/example_plugin/README.md

```diff
@@ -0,0 +1,22 @@
+# Example Plugin
+![example-plugin](example-plugin.gif)
+
+Example plugin provides an example usage of the ixbrl-viewer plugin infrastructure.
+
+## Highlighting
+The example plugin is able to highlight any element containing the letter 'T' with the color red.
+Click the settings and settings and select the toggle on and off to see it in action.
+
+## Running
+From the `example_plugin` directory
+
+```bash
+npm install
+npm run build
+```
+
+Then using arelle
+
+```bash
+python arelleCmdLine.py --plugins=<PATH_TO>/iXBRLViewerPlugin/__init__.py -f <FILING>.htm --save-viewer <VIEWER>.html --viewer-url <PATH_TO>/examples/example_plugin/dist/example-extended-ixbrl-viewer.js
+```
```

Comment:
I think it would be good to document the available extension methods.  These are currently listed in a comment at the top of `ixbrlviewer.js` so perhaps we could just add a note here pointing to that comment?

Reply:
I thought it was pretty well documented [here](https://github.com/Workiva/ixbrl-viewer/blob/master/PLUGINS.md#plugins) as long as thats up to date?

Reply:
Good point!  I'd completely forgotten about that!